### PR TITLE
refactor: make base argument compulsory in `math/base/special/floorsd`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   makes the base argument, `b` as compulsory in [`math/base/special/floorsd`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/floorsd).

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/floorsd/lib/main.js [hunk 1]
```diff
@@ -37,27 +37,26 @@ var exponent = require( '@stdlib/number/float64/base/exponent' );
...
-* var v = floorsd( 12368.0, 2 );
+* var v = floorsd( 12368.0, 2, 10 );
 * // returns 12000.0
 *
 * @example
 * var v = floorsd( 0.0313, 2, 2 );
 * // returns 0.03125
 */
 function floorsd( x, n, b ) {
-	var base;
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/math/base/special/floorsd/lib/main.js:60]: We don't need a separate variable for base now, we can directly use `b`.

**Review by Planeshifter — APPROVED** [review]
Thanks for making this change, @gunjjoshi!

**Comment by Planeshifter** [comment]
@gunjjoshi For future reference, updated the commit message to indicate that this change constitutes a breaking change. Thanks again for this PR!

**Comment by gunjjoshi** [comment]
@Planeshifter Great! From next time, I would too, keep this in mind and would write the commit message accordingly.

**Pull request merged**